src/pages/WorldEditor/WorldEditor.tsx

```diff
@@ -1,50 +1,75 @@
 import React, { useCallback } from "react";
 import { useHistory } from "react-router-dom";
-
-import { ADMIN_V3_WORLDS_BASE_URL } from "settings";
+import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
 
 import { WorldNavTab } from "types/world";
 
+import { adminWorldSpacesUrl } from "utils/url";
+
+import { useWorldBySlug } from "hooks/worlds/useWorldBySlug";
 import { useWorldParams } from "hooks/worlds/useWorldParams";
 
 import { WorldEditorAdvancedPanel } from "pages/WorldEditor/WorldEditorAdvancedPanel";
 import { WorldEditorEntrancePanel } from "pages/WorldEditor/WorldEditorEntrancePanel";
-import { WorldEditorStartPanel } from "pages/WorldEditor/WorldEditorStartPanel";
+import { WorldEditorGeneralPanel } from "pages/WorldEditor/WorldEditorGeneralPanel";
 
 import WithNavigationBar from "components/organisms/WithNavigationBar";
 
+import { AdminTitle } from "components/molecules/AdminTitle";
+import { AdminTitleBar } from "components/molecules/AdminTitleBar";
+import { LoadingPage } from "components/molecules/LoadingPage";
 import { WorldNav } from "components/molecules/WorldNav";
 
 import { AdminRestricted } from "components/atoms/AdminRestricted";
+import { ButtonNG } from "components/atoms/ButtonNG";
 
 import "./WorldEditor.scss";
 
 const PANEL_MAP = Object.freeze({
-  [WorldNavTab.start]: WorldEditorStartPanel,
+  [WorldNavTab.general]: WorldEditorGeneralPanel,
   [WorldNavTab.entrance]: WorldEditorEntrancePanel,
   [WorldNavTab.advanced]: WorldEditorAdvancedPanel,
 });
 
 export const WorldEditor: React.FC = () => {
   const history = useHistory();
   const { worldSlug, selectedTab } = useWorldParams();
+  const { world, isLoaded } = useWorldBySlug(worldSlug);
 
-  const navigateToHome = useCallback(
-    () => history.push(ADMIN_V3_WORLDS_BASE_URL),
-    [history]
-  );
+  const goBack = useCallback(() => {
+    if (history.length) {
+      history.goBack();
+    } else {
+      history.push(adminWorldSpacesUrl(world?.slug));
+    }
+  }, [history, world?.slug]);
```

Comment:
This is used when the cancel button is clicked. When someone cancels we generally want to back to where we've come from. However, if someone has opened this page inside a new tab then we need to go somewhere sensible. In this case, the dashboard for the world.

Reply:
> When someone cancels we generally want to back to where we've come from. 

Not really or not anymore. The preferred per IA is to remove the footer (and add a back button above the tab bar).

The cancel should take the user to a sensible place. Does it though? 

It might not mean "Go Back" since the Start form now renamed General isn't just for creating a new one, but also for editing an old one, thus a use case might be to advance to the Entrance settings, not bail out to worlds list.

Since I'm seeing a back button added in the title, instead of Approve, I will add a change request to just remove the `Footer` component from this editor and move the save/cancel buttons inside their respective forms.